Update the contributing.md (#763)
microsoft/QuantumKatas: .github/CONTRIBUTING.md

   * [Contributing new katas](#contributing-new-katas)
   * [Style guide](#style-guide)
   * [Updating the Katas to the new QDK version](#updating-the-katas-to-the-new-qdk-version)

>    * [Updating the Quickref](#updating-the-quickref)

Reviewer: Let's be a little formal here :-) I suspect not every contributor knows what "quickref" is, it's sort of hidden
```suggestion
   * [Updating the Q# Language Quick Reference](#updating-the-q-language-quick-reference)
```

After running this script you should validate that the update didn't introduce any breaking changes; see the next section for how to do this.

> ### Updating the Quickref

Reviewer: ```suggestion
### Updating the Q# Language Quick Reference
```

> You can update the quickref by making your changes in the `.tex` file. After your changes compile the `.tex` file into a `.pdf` file and create a PR with both files. An easy way to compile the `.tex` file is to use (portable) [TEXMAKER](https://www.xm1math.net/texmaker/download.html), this should directly find the needed packages to compline the `.tex` into a `.pdf`.

Reviewer: ```suggestion
You can update the Quick Reference by making your changes in the [`.tex` file](../quickref/qsharp-quick-reference.tex). After you do your changes, compile the `.tex` file into a `.pdf` file and include both files in the pull request. (The changes to the `.pdf` file will not be shown in the pull request, so be careful not to introduce any changes not included in the `.text` source!)

An easy way to compile the `.tex` file is to use (portable) [TEXMAKER](https://www.xm1math.net/texmaker/download.html), this should directly find the needed packages to compile the `.tex` into a `.pdf`.
```
